backend/src/main/java/com/rota/database/orm/engagement/Engagement.java

```diff
@@ -33,5 +35,6 @@
   private double hoursWorked;
 
   @ManyToOne(fetch = FetchType.LAZY)
+  @OnDelete(action = OnDeleteAction.CASCADE)
```

Comment:
I thought we wanted to keep the engagements if the staff was removed? Then again, the staff is never removed from the database, but made inactive instead, so probably irrelevant.

Reply:
Ah yeah I forgot that was a thing! The database tests contained a test where a staff member was removed and it was complaining about engagement foreign keys due to that staff member no longer existing.

Should I remove this and remove the test on removing staff?

Reply:
We should definitely discuss this in the call, I don't think our staff removal policy is 100% clear!